styles/styles.css

```
@@ -0,0 +1,316 @@
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("fonts/ptsans-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+body{
+    font-family: "PT Sans",sans-serif;
+    font-size: 18px;
+    line-height: 21px;    
+    background-color: #f2f2f2;
+}
+
+.visually-hidden{
+    visibility: hidden;
+}
```

На самом деле нужно ещё спрятать из потока, и отключить события мышки чтобы клики не перекрывали другие элементы
```suggestion
.visually-hidden {
  visibility: hidden;
  position: absolute;
  pointer-events: none;
}
```

Либо можно копипастнуть [готовый старый снипет](https://www.a11yproject.com/posts/how-to-hide-content/)
```
.visually-hidden {
  clip: rect(0 0 0 0);
  clip-path: inset(50%);
  height: 1px;
  overflow: hidden;
  position: absolute;
  white-space: nowrap;
  width: 1px;
}
```

---

Так же у тебя код не по стайлгайду академии. Надо будет поставить какой-то форматер